<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticketreturn</class>
 <widget class="QWidget" name="Ticketreturn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consult_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>咨询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>301</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>订单号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="package_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>您购买的套餐：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="order_number">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>是否退款？</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="returnSelect">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>价格：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="price">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="time">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>导游：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="guide">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
